dom.bs

```diff
@@ -906,8 +908,9 @@ correct defaults.</p>
  <li><p>Set <var>event</var>'s <a>initialized flag</a>.
 
  <li>
-  <p>Initialize <var>event</var>'s {{Event/timeStamp}} attribute to a {{DOMHighResTimeStamp}}
-  representing the high resolution time from the <a>time origin</a> to <var>time</var>.
+  <p>Initialize <var>event</var>'s {{Event/timeStamp}} attribute to the
+  <a>relative high resolution coarse time</a> given <var>time</var> and <var>event</var>'s
+  <a>relevant global object</a>.
 
   <p class=warning>User agents should set a minimum resolution of <var>event</var>'s
```

Comment:
It seems we  can remove this at which point the remaining p needs some rewrapping as it's the sole child of the li.

Reply:
Done